### PR TITLE
Yyin/feat/tdq 15048 international

### PR DESCRIPTION
**Pull request by yyin-talend**
TOP part finished.

### CODE REVIEW

File: main/plugins/org.talend.cwm.management/src/main/java/org/talend/cwm/management/i18n/messages.properties [hunk 1]
```diff
@@ -451,4 +453,12 @@ RepositoryNodeHelper.pattern.number=number
...
-RepositoryNodeHelper.Exchange=Exchange
\ No newline at end of file
+RepositoryNodeHelper.Exchange=Exchange
+DBColumnFolderRepNode.Columns=Columns ( {0} )
+DBTableFolderRepNode.Tables=Tables
+DBTableFolderRepNode.TablesWithCount=Tables ({0})
+DBViewFolderRepNode.Views=Views
+DBViewFolderRepNode.ViewsWithCount=Views ({0}).Tables=Tables
+DFColumnFolderRepNode.Columns=Columns ({0})
+ReportFolderRepNode
```
Comment (msjian) [inline main/plugins/org.talend.cwm.management/src/main/java/org/talend/cwm/management/i18n/messages.properties]: remove or add for  this line "ReportFolderRepNode"

File: main/plugins/org.talend.cwm.management/src/main/java/org/talend/cwm/management/i18n/messages.properties [hunk 2]
```diff
@@ -451,4 +453,12 @@ RepositoryNodeHelper.pattern.number=number
 RepositoryNodeHelper.pattern.phone=phone
 RepositoryNodeHelper.pattern.text=text
 RepositoryNodeHelper.sql.internet=internet
-RepositoryNodeHelper.Exchange=Exchange
\ No newline at end of file
+RepositoryNodeHelper.Exchange=Exchange
+DBColumnFolderRepNode.Columns=Columns ( {0} )
```
Comment (msjian) [inline main/plugins/org.talend.cwm.management/src/main/java/org/talend/cwm/management/i18n/messages.properties]: change "Columns ( {0} )" to Columns ({0})

File: main/plugins/org.talend.dataprofiler.core/src/messages.properties [hunk 3]
```diff
@@ -1472,4 +1472,79 @@ PatternSelectionForComponentDialog.title=Pattern Selector
...
+AbstractPagePart.unknown=unknown connection
+AnalysisColumnCompareTreeViewer.revertColumn=Reverse columns
+AnalysisColumnCompareTreeViewer.connectionNotAvaible=The connection is not available now!
+AnalysisTableTreeViewer.unknown=Unknown name
+AnalysisTableTreeViewer.unknownIndicator=Unknown indicator
+IndicatorsComp.unknownIndicator=Unknown indicator
+ParserRuleMasterDetailsPage.parserRule=Parser Rule
+ParserRuleMasterDetailsPage.typeRule=Type in the definition of your Parser Rules.
+ParserRuleMasterDetailsPage.addItem=Add New Item
+ParserRuleMasterDetailsPage.deleteItem=Delete Selcetion Item
```
Comment (msjian) [inline main/plugins/org.talend.dataprofiler.core/src/messages.properties]: Selcetion --> Selection

File: main/plugins/org.talend.dataprofiler.core/src/messages.properties [hunk 4]
```diff
@@ -1472,4 +1472,79 @@ PatternSelectionForComponentDialog.title=Pattern Selector
...
+ParserRuleMasterDetailsPage.moveupItem=Move Item Up
+ParserRuleMasterDetailsPage.moveDownItem=Move Item Down
+ParserRuleMasterDetailsPage.copyItem=Copy Selcetion Item
+ParserRuleMasterDetailsPage.parseItem=Paste Selcetion Item
```
Comment (msjian) [inline main/plugins/org.talend.dataprofiler.core/src/messages.properties]: the above two lines:  Selcetion --> Selection

File: main/plugins/org.talend.dataprofiler.core/src/messages.properties [hunk 5]
```diff
@@ -1472,4 +1472,79 @@ PatternSelectionForComponentDialog.title=Pattern Selector
...
+FileSystemImportWriter.VeriryVersion=Can't verify the imported version!
+ZipFileExportWriter.ExtensionInvalid=The root file extension is invalid!
+ExportForExchangeWizard.Title=Export For Talend Exchange
+ExportForExchangeWizard.ExportItem=Export Item
+ImportWizard.Title=Import Item
+ImportWizard.ImportItem=Import Item
+ExportWizard.Title=Export Item
+ExportWizard.Export=Export Item
+ExportWizardPage.CanNotFind=Can't find the file: {0}
+ExportWizardPage.missDepend= " {0} " miss dependency : {1}
```
Comment (msjian) [inline main/plugins/org.talend.dataprofiler.core/src/messages.properties]: change " {0} " to "{0}"

File: main/plugins/org.talend.dataprofiler.core/src/messages.properties [hunk 6]
```diff
@@ -1472,4 +1472,79 @@ PatternSelectionForComponentDialog.title=Pattern Selector
...
+IndicatorDefinitionMaterPage.highValue=highlighted values
+IndicatorDefinitionMaterPage.totalValue=total
+IndicatorDefinitionMaterPage.ganttChart=gantt chart functions
+IndicatorEditor.Indicator=Indicator Definition
+IndicatorEditor.UserIndicator=User Define Indicator Definition
+FileSystemExportWriter.Export=Exporting {0}
+FileSystemExportWriter.ExportFail=Export failed! {0} is not existed
+FileSystemExportWriter.RootNotExist=The root directory does not exist
+FileSystemImportWriter.CheckFailed=" {0} " check item conflict failed!
+FileSystemImportWriter.MissDepend=" {0} " missing dependented file : {1}
```
Comment (msjian) [inline main/plugins/org.talend.dataprofiler.core/src/messages.properties]: the above tow lines change " {0} " to "{0}"

File: main/plugins/org.talend.dataprofiler.core/src/org/talend/dataprofiler/core/ui/imex/model/FileSystemExportWriter.java [hunk 7]
```diff
@@ -115,7 +116,7 @@ public void write(ItemRecord[] records, IProgressMonitor monitor) {
 
                 Map<IPath, IPath> toImportMap = mapping(record);
 
-                monitor.subTask("Exporting " + record.getName());//$NON-NLS-1$ 
+                monitor.subTask(DefaultMessagesImpl.getString("Exporting ", record.getName()));//$NON-NLS-1$
```
Comment (msjian) [inline main/plugins/org.talend.dataprofiler.core/src/org/talend/dataprofiler/core/ui/imex/model/FileSystemExportWriter.java]: i think should change
DefaultMessagesImpl.getString("Exporting "
to
DefaultMessagesImpl.getString("FileSystemExportWriter.Export"

File: main/plugins/org.talend.dataprofiler.core/src/messages.properties [hunk 8]
```diff
@@ -1472,4 +1472,79 @@ PatternSelectionForComponentDialog.title=Pattern Selector
...
+ImportWizard.Title=Import Item
+ImportWizard.ImportItem=Import Item
+ExportWizard.Title=Export Item
+ExportWizard.Export=Export Item
+ExportWizardPage.CanNotFind=Can't find the file: {0}
+ExportWizardPage.missDepend= " {0} " miss dependency : {1}
+ChangePerspectiveAction.SwitchTo=Switch to {0}
+ChangePerspectiveAction.ShowFailed=Show perspective failed.
+RespositoryDetailView.Version=Version:
+FunctionDependencyWizard.Title=Just a test
```
Comment (msjian) [inline main/plugins/org.talend.dataprofiler.core/src/messages.properties]: I think we should change "Just a test" to "New Function Dependency Analysis"

Comment (yyin-talend) [reply main/plugins/org.talend.dataprofiler.core/src/messages.properties]: fixed all

**Pull request merged**